server/mappers/headerMapper.test.ts

```diff
@@ -30,13 +30,13 @@ describe('HeaderMapping', () => {
       expect(headerData.prisonerName).toBe('Saunders, John')
       expect(headerData.prisonerNumber).toBe('G6123VU')
       expect(headerData.prisonId).toBe('MDI')
-      expect(headerData['backLinkLabel']).not.toBeDefined()
-      expect(headerData['profileBannerTopLinks']).not.toBeDefined()
-      expect(headerData['alerts']).not.toBeDefined()
-      expect(headerData['tabLinks']).not.toBeDefined()
-      expect(headerData['photoType']).not.toBeDefined()
-      expect(headerData['restrictedPatient']).not.toBeDefined()
-      expect(headerData['hideBanner']).not.toBeDefined()
+      expect(headerData['backLinkLabel' as keyof typeof headerData]).not.toBeDefined()
+      expect(headerData['profileBannerTopLinks' as keyof typeof headerData]).not.toBeDefined()
+      expect(headerData['alerts' as keyof typeof headerData]).not.toBeDefined()
+      expect(headerData['tabLinks' as keyof typeof headerData]).not.toBeDefined()
+      expect(headerData['photoType' as keyof typeof headerData]).not.toBeDefined()
+      expect(headerData['restrictedPatient' as keyof typeof headerData]).not.toBeDefined()
+      expect(headerData['hideBanner' as keyof typeof headerData]).not.toBeDefined()
```

Comment:
Why can't these just use the dot syntax like the ones above?

Reply:
Yeah bit of a strange test... these fields don't exist on the object - I think it's trying to show that it's definitely not the other object that has these fields being returned 🤷